### ISSUE TITLE
Lottie animation doesn't work on android

### ISSUE THREAD

**Issue by jwirjono** [lottie-react-native/lottie-react-native#1247]
### Description

Hi there.

I am experiencing a problem with my animation, on iOS it works fine but on android the animation seems to run a bit weirdly. (It is supposed to run along the track) Not be reflected like in the video.

Animation Link: [https://app.lottiefiles.com/share/de484dcb-86d0-462e-9cc0-3cb84fadba03 2](https://app.lottiefiles.com/share/de484dcb-86d0-462e-9cc0-3cb84fadba03)

React Native version: 0.72.4
lottie react native version: 5.1.6

![image](https://github.com/lottie-react-native/lottie-react-native/assets/50869390/273a6b84-f0c9-4b8e-879b-dad9b5dfefec)

git : https://github.com/jwirjono/lottieAnimation

I’ve tried upgrading to lottie react native version 6.7.2. but this breaks it on iOS too and makes no difference to the android version.

### Steps to reproduce

Install the apps and then run the apps to check the animation

### Snack or a link to a repository

https://github.com/jwirjono/lottieAnimation

### Lottie React Native version

5.1.6

### React Native version

0.72.4

### Platforms

Android

### Workflow

React Native

### Architecture

Paper (Old Architecture)

### Build type

Debug app & dev bundle

### Device

Android emulator

### Acknowledgements

Yes

**Comment by matinzd** [comment]
Have you tried this animation on [Native Lottie app](https://play.google.com/store/apps/details?id=com.airbnb.lottie&hl=en) from Google Play? If it's broken there as well, you'll need to file an issue in [lottie-android](https://github.com/airbnb/lottie-android) repository.

**Comment by jwirjono** [comment]
Yes we have tried it on Native lottie app and it also broken there
